rest-api/src/test/java/gov/cms/qpp/conversion/api/integration/StorageServiceImplIntegrationTest.java

```diff
@@ -30,21 +45,10 @@
 import org.springframework.test.context.junit4.rules.SpringClassRule;
 import org.springframework.test.context.junit4.rules.SpringMethodRule;
 
-import javax.inject.Inject;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.lang.reflect.Field;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.TimeoutException;
-
-import static org.junit.Assert.fail;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.when;
-
 @SpringBootTest
 @RunWith(LocalstackTestRunner.class)
 @PropertySource("classpath:application.properties")
-public class StorageServiceImplIntegration {
+public class StorageServiceImplIntegrationTest {
```

Comment:
@omnivion The reason for the failing test is because you appended `Test` which causes Maven to pick it up and run it.  Sadly, our CircleCI environment isn't set-up with `make` or any of the other requirements of Localstack.  Hence, this test being disabled from running in CircleCI in `master`